examples/tests/run_all_tests.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+function section_header() {
+  printf "\n"
+  printf "╓─────[ $@ ]─────╖"
+  printf "\n"
+}
+
+RIG_DIR=../..
+TESTS_DIR=examples/tests
+cd "${RIG_DIR}"
+
+docker build -t accenture/reactive-interaction-gateway .
+
+cd "${TESTS_DIR}"
+
+section_header "Running functional test suite for Examples with no auth"
+docker rm -f rig || true
+docker run -d --name rig \
+-e EXTRACTORS='{"greeting":{"name":{"stable_field_index":1,"event":{"json_pointer":"/data/name"}}},"greeting.jwt":{"name":{"stable_field_index":1,"jwt":{"json_pointer":"/username"},"event":{"json_pointer":"/data/name"}}},"nope":{"fullname":{"stable_field_index":1,"jwt":{"json_pointer":"/fullname"},"event":{"json_pointer":"/data/fullname"}}},"example":{"email":{"stable_field_index":1,"event":{"json_pointer":"/data/email"}}}}' \
+-e JWT_SECRET_KEY=secret \
+-p 4000:4000 \
+accenture/reactive-interaction-gateway
+
+npm run cypress:run:noauth
```

Comment:
it would be nice (not mandatory) to fail right at the beginning in case cypress is not there cause `npm install` has not yet been invoked